packages/smart-contracts/src/contracts/ERC20TransferableReceivable.sol

```diff
@@ -183,60 +184,10 @@ contract ERC20TransferableReceivable is ERC721, ERC721Enumerable, ERC721URIStora
     receivableInfoMapping[currentReceivableTokenId] = ReceivableInfo({
       tokenAddress: erc20Addr,
       amount: amount,
```

Comment:
Comment:

I just realized that minters need to be very careful to ensure that the `tokenAddress` and `amount` provided when minting are correct and match the `owner` and `paymentReference`. If the minter uses a valid `owner` and `paymentReference` but accidentally uses an incorrect `tokenAddress` or `amount`, they would not be able to fix it. They'd have to start over with a new request before minting a new `TransferableReceivable` token.